GDP_analysis.Rmd

```diff
@@ -0,0 +1,95 @@
+---
+title: "GDP_Analysis"
+author: "Edward Lou"
+date: "2/21/2021"
+output: pdf_document
+---
+
+```{r setup, include=FALSE, warning=FALSE, message=FALSE}
+knitr::opts_chunk$set(echo = TRUE, warning=FALSE, message=FALSE)
+```
+
+```{r}
+# import libraries
+library(tidyverse)
+library(lubridate)
+library(ggpubr)
+library(dplyr)
+
+# load sixmonth data, omitting NA values
+sixmonth_data = na.omit(read.csv(".\\prepped_data\\sixmonth.csv", check.names = FALSE))
+
+# add gdp_pc quartile
+sixmonth_data$gdp_pc_quartile <- ntile(sixmonth_data$gdp_pc, 4)
+```
+\newpage
+```{r}
+# check correlation between gdp_pc and overall GHSI index
+ggscatter(sixmonth_data,x='gdp_pc' ,y='overall',
+          add='reg.line',cor.coef=TRUE,cor.method='pearson',
+          xlab='GDP per-capita',ylab='Overall GHSI Score')
+```
+\newpage
+```{r}
+# cases per-capita vs overall GHSI
```

Comment:
Can you turn some of these comments into headings/text that is rendered in the PDF. Hopefully once these are checked in we can read through the pdf and understand the whole process without needing to go back and forth with the source code

Reply:
I will reformat to make it more clear.

Reply:
Done.